students/skowronski_rafal/lesson_03_functions/calculator.py

```diff
@@ -0,0 +1,48 @@
+# Refactored version of 'Calculator' project
+
+
+def _execute(function):
+    operand_1 = int(input('Input 1st operand: '))
+    operand_2 = int(input('Input 2nd operand: '))
+
+    return function(operand_1, operand_2)
+
+
+def _help():
+    print('\nWelcome to the machine:')
+    print('a - add')
+    print('s - subtract')
+    print('m - multiply')
+    print('d - divide')
+    print('p - power')
+    print('h,? - help')
+    print('q - QUIT')
+
+
+def _exit():
+    print('GOOD BYE')
+    exit(0)
+
+
+def _main():
+
+    functions = {
```

Comment:
Good
You can use non lambda functions

def add(x, y):
    return x + y

functions = {
     'a':  add
}

but lamdbas are also good